clustered/integration-test/src/test/java/org/ehcache/clustered/writebehind/WriteBehindTestBase.java

```diff
@@ -102,4 +103,28 @@ PersistentCacheManager createCacheManager(URI clusterUri) {
       .withCache(testName.getMethodName(), cacheConfiguration)
       .build(true);
   }
+
+  PersistentCacheManager createCacheManagerWithLoaderWriterWithFailure(URI clusterUri) {
+    evenNumberLoaderWriter = new EvenNumberLoaderWriter();
+    CacheConfiguration<Long, String> cacheConfiguration =
+      newCacheConfigurationBuilder(Long.class, String.class, ResourcePoolsBuilder.newResourcePoolsBuilder()
```

Comment:
It might make sense to extract out the commonality between this method and the existing `createCacheManager(...)` method so that it's easier to see where the differences are.